pinot-common/src/test/java/org/apache/pinot/common/function/FunctionDefinitionRegistryTest.java

```diff
@@ -42,9 +42,9 @@ public class FunctionDefinitionRegistryTest {
   );
   private static final List<String> IGNORED_FUNCTION_NAMES = ImmutableList.of(
       // functions we are not supporting post transform anyway
-      "valuein", "mapvalue", "inidset", "lookup", "groovy", "scalar", "geotoh3", "case", "not_in", "timeconvert",
+      "valuein", "mapvalue", "inidset", "lookup", "groovy", "scalar", "geotoh3", "not_in", "timeconvert",
       // functions not needed for register b/c they are in std sql table or they will not be composed directly.
-      "in", "and", "or", "not", "range", "extract"
```

Comment:
why is "not" also included here?

Reply:
It looks like 'NOT' has been implemented and gets added to the function registry. So removed it from the ignored list for the unit test